Code/GraphMol/MolDraw2D/MolDraw2DDetails.h

```diff
@@ -56,9 +56,9 @@ RDKIT_MOLDRAW2D_EXPORT void arcPoints(const Point2D &cds1, const Point2D &cds2,
                                       std::vector<Point2D> &res,
                                       float startAng = 0, float extent = 360);
 
-//! add R/S, relative stereo, and E/Z annotations to atoms and bonds
-RDKIT_MOLDRAW2D_EXPORT void addStereoAnnotation(
-    const ROMol &mol, bool includeRelativeCIP = false);
+[[deprecated(
+    "please use Chirality::addStereoAnnotations instead")]] RDKIT_MOLDRAW2D_EXPORT void
+addStereoAnnotation(const ROMol &mol, bool includeRelativeCIP = false);
```

Comment:
Given the function is currently not in a public header or exposed via python, do you need to do this? Doesn't bother me either way, just curious why the backwards compatible API is needed.

Reply:
That's a good point. I didn't look to see if `MolDraw2DDetails.h` is actually installed.

Reply:
Cool, LGTM!